harvard-lts/CURIOSity
# Maintenance sprint a11y updates (multiple tickets)

**Maintenance sprint a11y updates (multiple tickets)**
* * *

**JIRA Ticket**:
* [LTSMAINT-852: Update accessibility links in footer](https://at-harvard.atlassian.net/browse/LTSMAINT-852)
* [LTSMAINT-853: Make pull quotes smaller](https://at-harvard.atlassian.net/browse/LTSMAINT-853)
* [LTSMAINT-854: Include alt-text/title for Harvard Library logo link](https://at-harvard.atlassian.net/browse/LTSMAINT-854)
* [LTSMAINT-855: Make IIIF logo in sidebar decorative](https://at-harvard.atlassian.net/browse/LTSMAINT-855)

# What does this Pull Request do?
Addresses multiple accessibility/usability updates as requested by product owner and users

# How should this be tested?

Spin up this branch locally and check for the following (broken down by ticket):
* LTSMAINT-852: Update accessibility links in footer
   * Library Accessibility” link above staff login that goes to this page links to [Harvard Library - Accessibility at Harvard Library](https://library.harvard.edu/accessibility)  
   * “Accessibility” link on the right side of the footer goes to [Home | University Disability Resources](https://accessibility.harvard.edu/)
   * No longer have social media icons in the bottom right
   * Copyright year in bottom right paragraph says "2024"
   * All of these changes should have it align with the Harvard Library website footer (with the exception of the list of nav links, they have different links in their footers)
* LTSMAINT-853: Make pull quotes smaller
   * New [pull quote style](https://localhost:21407/demo/feature/standard-widgets) when compared with [pull quote on prod ](https://curiosity.lib.harvard.edu/demo/feature/standard-item-widgets)is much smaller
* LTSMAINT-854: Include alt-text/title for Harvard Library logo link
   * Inspect the Harvard Library logos in the header and footer
   * Logo links will have a title that reads "Harvard Library website" (currently in prod, the header logo doesn't have a title attribute and the footer logo only says "Harvard Library" which is a little vague)
* LTSMAINT-855: Make IIIF logo in sidebar decorative
   * Navigate to any item detail page and inspect the IIIF logo in the right sidebar
   * The svg element now has a new attribute that reads `aria-hidden="true"`
   * Title for the svg element has also been updated to read "IIIF logo", which appears as a little popup when you hover over the logo (in prod, it just says "title")


# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? No
- integration tests? No

# Interested parties
@VanessaVenti @enriquediaz
